normandy/recipes/tests/test_api.py

```diff
@@ -275,6 +278,25 @@ def test_detail_view_includes_cache_headers(self, api_client):
         assert 'max-age=' in res['Cache-Control']
         assert 'public' in res['Cache-Control']
 
+    def test_midair_collisions(self, api_client):
+        recipe = RecipeFactory(name='first')
+        original_last_updated = recipe.last_updated
+
+        url = '/api/v1/recipe/{id}/'.format(id=recipe.id)
```

Comment:
Originally, we wanted to assert not only that the API worked as expected, but that the URLs were correct, since that is part of our public interface. I'm not entirely convinced we need to use that style everywhere though. Maybe we could have QA tests that validate the URLs, or just have a few tests that have hardcoded URLs, with comments as to why.